include/boost/astronomy/coordinate/base_differential.hpp

```diff
@@ -115,6 +115,17 @@ struct base_differential
     {
         return this->diff;
     }
+    //! returns the differential of calling object
+    bg::model::point
+    <
+        CoordinateType,
+        DimensionCount,
+        CoordinateSystem
+    >
+    get_point() const
```

Comment:
Function to return current differential of this object. By doing this we are able to calculate cross of differential objects.

Reply:
Look at https://github.com/BoostGSoC19/astronomy/pull/123#discussion_r407511280